README.md

```diff
@@ -747,7 +747,7 @@ As you can see all the classes in a class hierarchy actually share oneclass vari
   hash = { "one" => 1, "two" => 2, "three" => 3 }
```

Comment:
This section is about string keys, rather than spaces, so I'd go one step further and remove the spaces on this line as well.

And we can add a third example with a hash under the "No spaces after..." bit

Reply:
@fran-fac Pushed up a change for that now thanks, please let me know if OK